invenio_circulation/api.py

```diff
@@ -92,6 +95,41 @@ def __init__(self, data, model=None):
             finalize_event='save',
         )
 
+    @classmethod
+    def item_available(cls, item_pid):
```

Comment:
I think this should be just a func in the file `api.py` and not in the Loan class, I don't see why it should belong to Loan.

Reply:
done